packages/emitter/README.md

```diff
@@ -0,0 +1,12 @@
+# Emitter
+
+This utility function allows us to listen for changes to a store and then fire events when the store changes.
+
+### Usage
+
+```js
+import { emitter } from '@automattic/emitter';
+const Store = createAStore();
+emitter( Store );
```

Comment:
How do folks feel about this API where we mutate the object to add `EventEmitter` methods? `emitter( obj )` returns undefined, which may make it tricky to reason about (and makes it very difficult to type 🙂).